.travis.yml

```diff
@@ -22,6 +22,7 @@ cache:
     - ~/.cache
 before_install:
   - pip install --user awscli
+  - google-chrome-beta --headless --disable-gpu --remote-debugging-port=9222 http://localhost &
```

Comment:
For CI, the debugging port should be non-existent -- we don't even include the xdebug code in the docker image.

Reply:
Yeah, I was grasping at straws and pasting stuff there. I've already reverted that.

I'm still not sure why all our builds are failing, but was trying to see it was related to what this guy saw: https://travis-ci.community/t/cypress-chrome-all-build-broken-at-same-time/7166/2